packages/cli/src/admin-sdk.ts

```diff
@@ -0,0 +1,224 @@
+import { createClient } from './autogen/client/createClient';
+import fse from 'fs-extra';
+import { kebabCase, omit } from 'lodash';
+import chalk from 'chalk';
+import { readAsJson, getFiles, getDirectories, replaceField } from './utils';
+import cliProgress from 'cli-progress';
+import { createHash } from 'crypto';
+import traverse from 'traverse';
+
+const multibar = new cliProgress.MultiBar(
+  {
+    clearOnComplete: false,
+    hideCursor: true,
+    format: '|{bar}| {name} | {value}/{total}',
+  },
+  cliProgress.Presets.shades_grey
+);
+
+// todo change this to prod
+const root = 'https://qa.builder.io';
```

Comment:
```suggestion
const root = process.env.ROOT_URL || 'https://qa.builder.io';
```

This way the root URL can be specified at any time at run/build time, and fallback to QA if it's not present.

Like if we do (example):
```
env ROOT_URL='https://production.builder.io' npm publish
```
in a Github action, published artifacts will always point to "production", regardless of what we use during development when running `npm run whatevs`. That also allows devs to run one-off commands against any environment, without having to change any code.

To make managing env vars easier during development, for example we could use [`direnv`](https://formulae.brew.sh/formula/direnv).

Reply:
same as here https://github.com/BuilderIO/builder/pull/321#discussion_r549832967